components/bl-video-recorder-component/README.md

```diff
@@ -0,0 +1,41 @@
+# Video Recorder
+
+Video Recorder is a component of Backendless UI-Builder designer. This allows to record screen and use the recorded file.
+
+
+<p align="center">
+  <img src="./thumbnail.png" alt="main thumbnail" width="780"/>
+</p>
+
+## Properties
+
+| Property    | Type       | Default Value    | Logic           | Data Binding | UI Setting | Description                                                  |
+|-------------|------------|------------------|-----------------|--------------|------------|--------------------------------------------------------------|
+| Width       | *Number*   | 720              |                 | NO           | YES        | Controls width of component, in 'px'.                        |
+| Height      | *Number*   | 480              |                 | NO           | YES        | Controls height of component, in 'px'.                       |
+| Allow Audio | *Checkbox* | `true`           |                 | NO           | YES        | Controls allow recording audio from screen, page, etc.       |
+| File Name   | *Text*     | "Recorded Video" | File Name Logic | YES          | YES        | This is a handler to control the name of the downloaded file.|
+
+## Events
+
+| Name               | Triggers                                  | Context Blocks                            |
+|--------------------|-------------------------------------------|-------------------------------------------|
+| On Start Recording | When the recording is started             |                                           |
+| On Stop Recording  | When the recording is stopped             |                                           |
+| On Download file   | When the recorded file starts downloading | Blob File: `{size: Number, type: String}` |
```

Comment:
This blob is not the same as at 34 line?

Also, will be better to add link for MDN than use this signature

Reply:
The same blob.